helpers/control.py

```diff
@@ -492,23 +492,32 @@ def __servers_pool_size(n_servers):
     return min(n_servers, MAX_THREADS)
 
 def servers_start(servers):
-    threads = __servers_pool_size(len(servers))
-    pool = multiprocessing.Pool(threads)
-    pool.map(Nginx.start, servers)
+    for server in servers:
+        try:
+            server.start()
+        except Exception as e:
```

Comment:
If exception is raised during `server.start()`, its logged and nothing more. The test case won't be aware of startup issues and the test will continue to work. This seems to be incorrect behaviour.

Reply:
No. It will also be state == ERROR for this server. And teardown looks for states after stopping all servers

Reply:
It will be also state == ERROR

Reply:
But in start and get_stats raise exception is a good idea, fixed